doc/sphinx/source/utils/RTW/add_a_recipe.rst

```diff
@@ -76,47 +72,41 @@ How to add a recipe to the |RTW|
 
     cylc stop recipe_test_workflow/*
 
-#. Run the |RTW|, as detailed in the
-   :doc:`Quick Start Guide <user_guide/quick_start>`; it is expected that the
-   ``compare`` task will fail.
+#. Run the |RTW|, as detailed in the :ref:`quick_start_guide`; it is expected
+   that the ``compare`` task will fail.
 
 #. Update the Known Good Outputs (|KGOs|):
 
    * Recursively copy the recipe output directory (i.e.
-     ``recipe_<recipe>_<date>_<time>``) from the
-     ``${HOME}/cylc-run/recipe_test_workflow/run1/share/cycle/<cycle>``
-     directory to your site-specific KGO directory, as detailed by the
+     ``recipe_<recipe>_<date>_<time>/``) from the
+     ``${HOME}/cylc-run/recipe_test_workflow/run1/share/cycle/<cycle>/``
+     directory to your site-specific |KGO| directory, as detailed by the
      ``KGO_ROOT_PATH`` option in the site-specific ``.conf`` file in the
-     ``esmvaltool/utils/recipe_test_workflow/recipe_test_workflow/opt/``
-     directory::
+     ``esmvaltool/utils/recipe_test_workflow/opt/`` directory::
 
        cp -r ${HOME}/cylc-run/recipe_test_workflow/run1/share/cycle/<cycle>/recipe_<recipe>_<date>_<time> <KGO_ROOT_PATH>
```

Comment:
```suggestion
       cp -r ${HOME}/cylc-run/recipe_test_workflow/runN/share/cycle/<cycle>/recipe_<recipe>_<date>_<time> <KGO_ROOT_PATH>
```
If you agree with the above `runN` suggestion, it should be updated here also.

Reply:
Thanks @alistairsellar! Done in ea4257713 😊